gengo/languages.yaml

```
@@ -55,6 +55,8 @@
   matchers:
     extensions:
       - cs
+      - csx
+      - cake
```

```suggestion
      - cake
      - cs
      - csx
```
The alphabetization might be a bit annoying now, but I'm hoping it makes it easier to find things once this file gets really large.